### PR TITLE
Fixed api v2 documentation after proofread

### PR DESCRIPTION
**Pull request by meganelacheny**

### CODE REVIEW

File: source/includes/v2/_odsql.md [hunk 1]
```diff
@@ -4,29 +4,43 @@ Filtering features are built in the core of OpenDataSoft API engine.
...
+- the `select` clause: it allows to choose the returned fields, to give them an alias, or to manipulate them with functions like count, sum, min, max, etc.
+- the `where` clause: it acts as a filter for the returned datasets or records, thanks to boolean operations, filter functions, arithmetic expressions, etc.
+- the `group by` clause: it allows to aggregate rows together based on fields, numeric ranges, or dates
+- the `order by` and `limit` clauses: they allow to choose the order and quantity of rows received as a response
 
-These clauses are used as parameters in the Search API v2 for searching, aggregating and exporting datasets and records. Depending on the endpoint you use, some features of the query language are available or not in the request.
+These clauses are used as parameters in the Search API v2 for searching, aggregating and exporting datasets and records. Depending on the used endpoint, some features of the query language are available or not in the request.
+
+<aside>
+The whole query language is case insensitive and spaced are optional. In this documentation, upper case will however be used for language keywords, only for clarity purpose.
```
Comment (vbarbaresi) [inline source/includes/v2/_odsql.md]: spaces*

File: source/includes/v2/_odsql.md [hunk 2]
```diff
@@ -94,10 +109,10 @@ my_boolean_field is TRUE
 my_boolean_field: FALSE
 ```
 
-Boolean literals should be used in boolean filters and can be either TRUE or FALSE keywords (non case sensitive)
+A boolean literal can either be a TRUE or a FALSE keyword (non case sensitive). It should be used in boolean filters.
```
Comment (vbarbaresi) [inline source/includes/v2/_odsql.md]: non case sensitive = case insensitive?

File: source/includes/v2/_odsql.md [hunk 3]
```diff
@@ -106,40 +121,43 @@ distance(my_geo_field, geom'POINT(1 1)', 10km)
 geometry(my_geo_field, geom'{"type": "Polygon","coordinates":[[[100.0, 0.0],[101.0, 0.0],[101.0, 1.0],[100.0, 1.0],[100.0,0.0]]]}')
 ```
 
-Geometry literals are defined with a geom keyword followed by a valid geometry expression enclosed in single quotes. Supported geometry expression are:
+A geometry literal is defined with a geom keyword followed by a valid geometry expression enclosed in single quotes.
+
+Supported geometry expression are:
```
Comment (vbarbaresi) [inline source/includes/v2/_odsql.md]: expressions*

File: source/includes/v2/_odsql.md [hunk 4]
```diff
@@ -178,19 +196,28 @@ List of reserved keywords:
...
 - year
 
+
 ## Select clause
 
-The select clause can be used in records search APIs as the parameter select. Its goal is to allow you to choose the fields that will be returned for each row, transform them using arithmetic, rename them, add computed virtual fields, include or exclude fields based on a pattern.
+The select clause can be used in records search APIs as the parameter `select`.
+
+The select clause allows to:
+- allow to choose the fields that will be returned for each row
```
Comment (vbarbaresi) [inline source/includes/v2/_odsql.md]: double allow syntax is weird (you already said "allows" in the line above)

File: source/includes/v2/_odsql.md [hunk 5]
```diff
@@ -352,8 +437,18 @@ DISTANCE(field_name, GEOM'<geometry>', 1 km)
 DISTANCE(field_name, GEOM'<geometry>', 100 yd)
 ```
 
-The distance function limits the result set to a geographical area defined by a circle defined by its center and a distance. The center of the circle is expressed as a [geometry literal](#geometry-literal)). The distance is numeric and can have a unit in mi, yd, ft, m, cm, km, mm.
+The distance function limits the result set to a geographical area defined by a circle. This circle must be defined by its center and a distance.
 
+- The center of the circle is expressed as a [geometry literal](#geometry-literal)).
+- The distance is numeric and can have a unit in:
```
Comment (vbarbaresi) [inline source/includes/v2/_odsql.md]: an unit

File: source/includes/v2/_odsql.md [hunk 6]
```diff
@@ -433,69 +581,13 @@ numeric_field: [1..10]   # Use `:` instead of `IN` operator
...
-```plain-text
-"tree"
-"tree" AND "flower"
-"tree" OR "car"
-NOT "dog"
-"dog" AND NOT "cat"
-```
-
-> Examples of search query with mutliple words
+In filters results are based on a numeric or a date range. In filters only work on a field literal.
```
Comment (vbarbaresi) [inline source/includes/v2/_odsql.md]: In the second phrase, it looks like you accidentally a word

File: source/includes/v2/_odsql.md (hunk 2 above)
Comment (clement-tourriere) [reply source/includes/v2/_odsql.md]: Yes, but this is because I am not an insensitive person

**Pull request merged**